omod/src/main/java/org/openmrs/module/fhir/swagger/codegen/SwaggerCodeGenerator.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ * <p/>
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ * <p/>
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.fhir.swagger.codegen;
+
+import io.swagger.codegen.ClientOptInput;
+import io.swagger.codegen.DefaultGenerator;
+import io.swagger.codegen.config.CodegenConfigurator;
+import io.swagger.models.Swagger;
+import io.swagger.parser.SwaggerParser;
+import io.swagger.util.Json;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.fhir.exception.FHIRModuleOmodException;
+import org.openmrs.module.fhir.swagger.utils.Utils;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SwaggerCodeGenerator {
+
+
+    protected Log log = LogFactory.getLog(getClass());
+    private Map<String, String> sdkLanguages = new HashMap<>();
+    private static final String ZIP_ARCHIVE_NAME_PREFIX = "OPENMRS-FHIR-CLIENT";
+    public static final String JSON_EXTENSION = ".json";
+
+
+    public SwaggerCodeGenerator() {
+        //Supported SDK languages
+        sdkLanguages.put("java", "io.swagger.codegen.languages.JavaClientCodegen");
```

Comment:
are we only supports these languages only? I thought according to the discussion we are supporting all available languages. Can you please check